src/components/Chip.jsx

```diff
@@ -2,9 +2,9 @@ import { useRouter } from "next/router";
 
 export const ChipTypeStyles = {
   Tag: "border-neutral-chip-shade bg-neutral-chip",
-  "High Concern": "border-high-concern-shade bg-high-concern",
-  "Some Concern": "border-some-concern-shade bg-some-concern",
-  "No Concern": "border-no-concern-shade bg-no-concern",
+  HighConcern: "border-high-concern-shade bg-high-concern",
```

Comment:
check if this is responsible for lack of chip colors on the deploy preview for this pr https://deploy-preview-54--canine-assistants.netlify.app/

Reply:
fixed! (it was a funky string matching thing)